### PR TITLE
update smb multi join code gen to the latest

### PR DESCRIPTION
**Pull request by farzad-sedghi**
updated the code generator for `SMBMultiJoin.scala` to reflect the latest manual changes

### CODE REVIEW

File: scripts/smb_multijoin.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def mkCogroupFnRetVal(n, aWrapper=None, otherWrapper=None):
 
 def mkTransformFnRetVal(n, aWrapper=None, otherWrapper=None):
     vals = fnRetValHelper(n, aWrapper, otherWrapper)
-    return 'SortedBucketScioContext#SortMergeTransformReadBuilder[KEY, (%s)]' % ', '.join(vals)
+    return 'SortedBucketScioContext#SortMergeTransformReadBuilder[KEY, KEY, Void, (%s)]' % ', '.join(vals)
```
Comment (shnapz) [inline scripts/smb_multijoin.py:60]: was this call outdated?

**Review by RustedBones — APPROVED** [review]
LGTM. is running the script creating any diff ?

File: scripts/smb_multijoin.py (hunk 1 above)
Comment (farzad-sedghi) [reply scripts/smb_multijoin.py:60]: this is what was changed manually when adding the secondary SMB sort key I believe

**Comment by farzad-sedghi** [comment]
> LGTM. is running the script creating any diff ?

no after formatting both sides have zero diffs

**Pull request merged**